js/load.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+(function () {
+  var URL = 'https://js.dump.academy/kekstagram/data';
+  var TIMEOUT_IN_MS = 10000;
+  var StatusCode = {
+    OK: 200
+  };
+  var arrayPictures = [];
```

Comment:
```suggestion
  var pictures = [];
```
Нужно исправить название и перенести в модуль `gallery`.